lib/manageiq/api/client.rb

```diff
@@ -1,9 +1,29 @@
+require "active_support"
+require "active_support/core_ext"
+require "faraday"
+require "faraday_middleware"
+require "json"
+require "more_core_extensions/all"
+
+require "manageiq/api/client/action"
+require "manageiq/api/client/api_version"
+require "manageiq/api/client/authentication"
+require "manageiq/api/client/collection"
+require "manageiq/api/client/connection"
+require "manageiq/api/client/error"
+require "manageiq/api/client/identity"
+require "manageiq/api/client/resource"
+require "manageiq/api/client/server"
+require "manageiq/api/client/server_api"
 require "manageiq/api/client/version"
 
 module ManageIQ
   module API
     module Client
-      # Your code goes here...
+      def new(options = {})
+        Server.new(options)
+      end
+      module_function :new
```

Comment:
Prefer just doing `def self.new`.  However, I would *not* have a `new` method, as it complicates the code, because `new` implies to the User that it's being called on a class.  I would prefer `.connect` or something similar.   Or just get rid of the Server class and make Client *be* that class.